src/systems/mod.rs

```diff
@@ -21,17 +21,20 @@ mod spaceship_movement;
 mod stat_tracker;
 mod status_bar;
 mod store;
+mod timelimit;
 
 pub use self::{
     animation::AnimationSystem, blast::BlastSystem, boss::BossSystem,
     collision_detection::CollisionDetectionSystem, collision_handler::CollisionHandlerSystem,
     consumable::ConsumableSystem, defense::DefenseSystem, enemy::EnemySystem,
-    enemy_hit::EnemyHitSystem, enemy_spawn::SpawnerSystem, timelimit::TimeLimitSystem,
-    gamemaster::GameMasterSystem, item::ItemSystem, planets::PlanetsSystem,
+    enemy_hit::EnemyHitSystem, enemy_spawn::SpawnerSystem, gamemaster::GameMasterSystem,
+    hitbox_system::HitboxSystem, item::ItemSystem, planets::PlanetsSystem,
     player_hit::PlayerHitSystem, spaceship::SpaceshipSystem,
     spaceship_movement::SpaceshipMovementSystem, stat_tracker::StatTrackerSystem,
-    status_bar::StatusBarSystem, store::StoreSystem,
+    status_bar::StatusBarSystem, store::StoreSystem, timelimit::TimeLimitSystem,
 };
+
+// phase this out and instead use hitbox component
```

Comment:
nit: could we add "TODO" here? 
```suggestion
// TODO: phase this out and instead use hitbox component
```